Makefile

```diff
@@ -105,8 +105,6 @@ go-test-api: check-env
 mod-check: check-env
 	@echo 'running: go mod verify'
 	@go mod verify && [ "$(shell sha512sum go.mod)" = "`sha512sum go.mod`" ] || ( echo "ERROR: go.mod was modified by 'go mod verify'" && false )
-	@echo 'running: go list -mod=readonly -m all'
-	@go list -mod=readonly -m all 1> /dev/null
```

Comment:
This is not failing in most of the cases as of today except this PR and even i checkout the PR and if i add `go 1.22.0` and run mode-check its not failing as well.

Reply:
Do you know why this needs to be run?

Reply:
it was added as as part of https://github.com/ceph/ceph-csi/pull/3664

Reply:
dropped it again.